crates/server/src/services/verifier/verify_number.rs

```diff
@@ -142,10 +142,18 @@ impl Handler<Verify> for VerifierService {
             }
         }
 
-        // todo: generate scale-encoded signed data to be included in signup tx
+        let phone_number_hash =
+            sp_core::hashing::blake2_512(user_data.phone_number.clone().as_bytes());
+        let verification_evidence = sp_rpc::verifier::VerificationEvidence {
+            verifier_public_key: self.key_pair.unwrap().public(),
+            account_id: account_id,
+            username: user_data.user_name,
+            phone_number_hash: phone_number_hash,
+        };
+        let bytes = verification_evidence.encode();
 
         let response = VerifyNumberResponse {
-            data: vec![],
+            data: self.key_pair.unwrap().sign(&bytes).0.to_vec(),
```

Comment:
@HolyGrease  - don't we need to return the data and the signature as separate field so both can be included in the user's tx?

Reply:
When client call verify number method that means that client already has arguments, so it can use them to create transaction, so why we need to return them back?

Reply:
So client now create same structure it sent verifier and adds it to the tx and the verifier signature would check on that structure?

Reply:
I guess it is okay. Going to merge this PR and all related ones. Good job on it!